### DISCUSSION TITLE
Using `<ctrl-e>` and `<ctrl-y>`

### DISCUSSION THREAD

**Discussion by ogirginc** [nvim-tree/nvim-tree.lua#1352] [Q&A] [answered]
How can I revert `<ctrl-e>` to use it like a regular Neovim line up? The same goes for `<ctrl-y>` and use it as line down. Is it possible? 🤔

**Comment by kyazdani42** [comment, marked as answer]
set `key = "<C-e>", action = ""` in your mapping config

**Reply by ogirginc** [reply]
Thank you! 🙌 

Does this mean setting an action to empty string allows keys to use default functionality of the all existing key bindings on `nvim-tree`?

**Reply by alex-courtis** [reply]
> Does this mean setting an action to empty string allows keys to use default functionality of the all existing key bindings on `nvim-tree`?

It clears just that one `key` and removes that default binding. See `:help nvim-tree-mappings`
